utest/shared/src/main/scala/utest/framework/Formatter.scala

```diff
@@ -16,7 +16,10 @@ trait Formatter {
   def formatTruncateHeight: Int = 15
   def formatWrapWidth: Int = 100
 
-  def formatValue(x: Any) = testValueColor(x.toString)
+  def formatValue(x: Any) = {
+    val v = if (x != null) x.toString else ""
```

Comment:
@piotrkwiecinski what do you think about just making this `"" + x`? I think if you return `null` from a test case, we should print it. If you want to print nothing, you can already return `(): Unit`